web-service/helpers.py

```diff
@@ -226,7 +230,7 @@ def StoreInvalidUrl(siteInfo, url):
 
 ################################################################################
 
-def StoreUrl(siteInfo, url, final_url, real_final_url, content, encoding):
+def StoreUrl(siteInfo, url, content, encoding):
```

Comment:
A note on these values:
`url` was the original uribeacon url
`final_url` was the first redirected value
`real_final_url` was the final redirected value

Now, `url` is just the final redirected value.  We only cache that, no matter which initial uribeacon url it came from.

I'm not 100% sure if its possible for the server to know where it was redirected-from and return a different response based on that.  I looked up `referer` and that is *not* used to give this info.  So I think caching final url regardless of the path it took to get there is fine.